package.json

```diff
@@ -21,6 +21,7 @@
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.1.1",
     "prism-react-renderer": "^1.3.1",
+    "raw-loader": "^4.0.2",
```

Comment:
This seems to be deprecated in favour of webpack 5 native features. Is docusaurus already using v5? If so could we use that over the raw-loader package?

Reply:
Good point, I will look into that in a followup PR.